### ISSUE TITLE
[BUG - Develop] Unreliable EvMenu option offset in the webclient

### ISSUE THREAD

**Issue by InspectorCaracal** [evennia/evennia#2677]
#### Describe the bug
When viewing an EvMenu in the web client, the first line of the options is offset - in my case, to the right by one space. I tested it in the web client using the current versions of Firefox and Edge, as well as in my non-web mu* clients. The issue occurred only in the web client and appeared the same in both browsers.

#### To Reproduce
1. Create an EvMenu node with options
2. Log in using the web client
3. Access the EvMenu in game

#### Expected behavior
The option lines should be lined up evenly, as they are using a non-web client.

#### Develop-branch commit
473f2e379

**Comment by InspectorCaracal** [comment]
I went deeper into my character creation menu testing things out and discovered that the web client doesn't _always_ offset the first line. The nodes that have the offset versus the ones that don't are consistent - it doesn't change which are which if I navigate back and forth between them - but I can't see any rhyme or reason to it.

Here's a node that has the offset:
```python
def menunode_welcome(caller):
	"""Starting page."""
	text = dedent("""\
		|wWelcome to |mSunnyhill|w, the slice-of-life fantasy game.|n

		The character creation process is somewhat long, but if you have to
		stop or are interrupted/disconnected, you'll be able to resume from
		where you left off by using 'charcreate' with the same character
		name.

		If you need to take a break, enter 'q' to quit. Or, if you need more
		information, try entering 'help'.

		Before you begin, you must choose your |corigin|n. Choose an option
		below for a brief overview of its background and game mechanics.
	""")
	help = fill("In the world of |mSunnyhill|n, there are many different types "
				"of people. Due to the unique nature of the city, you can find "
				"examples of people who hail from virtually anywhere else in "
				"the world.")
	options = []
	options.append({"desc": "Learn about |cDragons|n", "goto": "menunode_dragon_info"})
	options.append({"desc": "Learn about |cElves|n", "goto": "menunode_elf_info"})
	options.append({"desc": "Learn about |cPixies|n", "goto": "menunode_pixie_info"})
	options.append({"desc": "Learn about |cShifts|n", "goto": "menunode_shifter_info"})
	return (text, help), options
```

And here is one that does _not_ have the offset:
```python
def menunode_shifter_info(caller):
	"""Information on playing a Shifter."""
	text = dedent("""\
		|bShifts|n

		Unlike the other three groups, |cShifts|n have little to no cultural
		identity of their own - even the name |wshift|n or |wshifter|n is a
		nickname they picked up due to their natural ability to shape-shift.
		
		Individuals or very small groups will typically spend their lives as
		wanderers, travelling from one society or culture to another, until
		deciding to settle and integrate into one of them. The next generation
		can then choose to stay with the society they were raised in, or to
		pick up and travel themselves.
		
		Their natural ability to shape-shift allows them to change their
		appearance to match one of the other groups - mostly. Except for the
		most highly skilled, all shifters have two "tells" - their ears,
		and their tail.
		
		Physically, shifters are born with a natural animal form. Aside
		from being mammalian, rarely herbivorous, and typically similar
		to their parents, the animal form any individal is born with can
		vary greatly. The natural size of a shifter doesn't seem to
		limit the size they can shift to.
	""")
	help = fill("If you're thinking of playing a Shifter, make sure "
				"to look at the other origins to decide which of them "
				"you were born in.")
	options = []
	options.append({"desc": "Become a |cShifter|n", "goto": "menunode_shift_form"})
	options.append({"desc": "Learn about |cDragons|n", "goto": "menunode_dragon_info"})
	options.append({"desc": "Learn about |cElves|n", "goto": "menunode_elf_info"})
	options.append({"desc": "Learn about |cPixies|n", "goto": "menunode_pixie_info"})
	return (text, help), options
```

I left all the flavor text in the pastes, just in case it makes a difference somehow.

**Comment by InspectorCaracal** [comment]
And _here_ is a node where the first line is lined up with the left edge as it should be, but the rest of the option lines are offset to the right instead....

```python
def menunode_feature(caller, raw_string, **kwargs):
	"""Setting a physical feature's description."""
	feature = kwargs['feature']
	try:
		feature_desc = caller.new_char.db.features[feature]
	except KeyError:
		feature_desc = None
	if feature_desc:
		text = "Your current |c%s|n description: |w%s|n\n\n" % ( feature, feature_desc )
	else:
		text = "You are describing |c%s|n.\n\n" % feature
	text += "Enter your %s description (or leave blank to leave unchanged):" % feature
	help = fill("Enter a description for your selected feature, e.g. "
	            "'a pair of large spiral horns, gleaming black' "
	            "or 'pale hair'."
				"You can also use any pronoun markers to self-reference in "
				"your feature, as well as ANSI color codes. For more info, "
				"check out the help files for 'ANSI' and 'pronouns'.")
	options = ( {"key": "_default", "goto": (_set_feature, kwargs)} )
	return (text, help), options
```

**Comment by InspectorCaracal** [comment]
I did a little poking around and it seems to be [this](https://github.com/evennia/evennia/blob/f093c8bcb1321a44e33003cf91cf9f565e028de7/evennia/utils/text2html.py#L388) incorrectly adding additional spacing before the color `<span>`. Commenting out that line solves the problem and appears to have no negative side-effects in regards to other formatting, although I didn't test it very rigorously.

**Comment by Griatch** [comment]
Closed in #2711
